src/encoder/encoderfdkaac.cpp

```diff
@@ -40,6 +40,7 @@ EncoderFdkAac::EncoderFdkAac(EncoderCallback* pCallback)
     QString libname = "";
 #ifdef __LINUX__
     libnames << "fdk-aac";
+    libnames << "libfdk-aac.so.1";
```

Comment:
We are using version 2 in todays modern Linux..
Is this meant to be dlopened ?

Reply:
The latest Ubuntu 20.4 LTS Focal has libfdk-aac.so.1 and no symlink to libfdk-aac.so

Ubuntu Groovy has a symlink to libfdk-aac.so.2.0.1 which should be picked first. I have not tested if libfdk-aac.so.2 actually works. Does it?

Reply:
The un-versioned symlink (.so) shouldn't be installed on end-users systems, only if the -dev packages are expected on developer ones.
I'm not sure which symbols broke.

Reply:
OK, now I try load in the following order so so.2 so.1 
I have compared the API and the used functions are unchanged.